controller.go

```diff
@@ -203,7 +200,7 @@ func New(cfgOptions ...config.Option) (NetworkController, error) {
 		return nil, err
 	}
 
-	drvRegistry, err := drvregistry.New(c.getStore(datastore.LocalScope), c.getStore(datastore.GlobalScope), c.RegisterDriver, nil, c.cfg.PluginGetter)
+	drvRegistry, err := drvregistry.New(c.getStore(datastore.LocalScope), c.getStore(datastore.GlobalScope), nil, nil, nil)
```

Comment:
I think this change is not necessary and causes troubles. I have issues where tests in moby cannot start network plugins and I guess it comes from there and with change below